src/submission-service.ts

```diff
@@ -120,10 +108,15 @@ async function processPendingQueue({
         'SUBMIT_STARTED',
       )
 
-      const submission = await prepareSubmissionData(formSubmission)
+      const preparedSubmission = await prepareSubmissionData(
+        pendingQueueSubmission,
+      )
+      pendingQueueSubmission.submission = {
+        ...preparedSubmission,
+      }
```

Comment:
```suggestion
      pendingQueueSubmission.submission = await prepareSubmissionData(
        pendingQueueSubmission,
      )
```

Reply:
Need to create a new `submission` object to get react to update when `submission` changes.